lib/screens/v2/candidate_generator.ex

```diff
@@ -3,7 +3,7 @@ defmodule Screens.V2.CandidateGenerator do
 
   alias Screens.V2.ScreenData
 
-  @callback candidate_templates() :: Screens.V2.Template.template()
+  @callback candidate_template() :: Screens.V2.Template.template()
```

Comment:
I don't want to make you do another rename, but is it still a candidate if there's only one and it's always used? Maybe `screen_template`?

Reply:
Good point, made the change.